app/src/main/java/org/standardnotes/notes/MainActivity.kt

```diff
@@ -33,18 +49,51 @@ class MainActivity : AppCompatActivity() {
 
     override fun onCreateOptionsMenu(menu: Menu): Boolean {
         super.onCreateOptionsMenu(menu)
-        val item = menu.add(getString(R.string.action_logout))
-        item.setIcon(android.R.drawable.ic_menu_search)
-        item.setShowAsAction(MenuItem.SHOW_AS_ACTION_NEVER)
+        menuInflater.inflate(R.menu.logged_in, menu)
         return true
     }
 
+    override fun onPrepareOptionsMenu(menu: Menu?): Boolean {
```

Comment:
eh, I don't think that's a problem for now. should be relatively easy to pull out logic later on.

plus, the app is pretty much working, doesn't hurt to do things more iteratively